CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ FEATURES:
 * Add destination and association resources to support Secrets Sync. Requires Vault 1.16+ ([#2098](https://github.com/hashicorp/terraform-provider-vault/pull/2098)).
 * Add support for configuration of plugin WIF to the AWS Secret Backend. Requires Vault 1.16+ ([#2138](https://github.com/hashicorp/terraform-provider-vault/pull/2138)).
 * Add support for Oracle database plugin configuration options `split_statements` and `disconnect_sessions`: ([#2085](https://github.com/hashicorp/terraform-provider-vault/pull/2085))
+* Improve logging to track full API exchanges between the provider and Vault ([#2139](https://github.com/hashicorp/terraform-provider-vault/pull/2139))
```

Comment:
We will need to rebase on main since v3.25.0 has already been released and this file is out of date.

Reply:
My bad! I guess slow and steady doesn't always win the race after all - Rebased and updated the README.md, hopefully that resolves any remaining issues - let me know if there's anything I've missed!